socs/agents/hwp_supervisor/agent.py

```diff
@@ -1457,6 +1528,23 @@ def make_parser(parser=None):
         '--gripper-iboot-outlets', nargs='+', type=int,
         help="Outlets for gripper iboot power")
 
+    pgroup.add_argument(
+        '--acu-instance-id', help="Instance ID for the ACU agent"
+    )
```

Comment:
Somewhere, maybe many places, you need to mention that passing this is necessary for the ACU check to be active.  I.e. it's not sufficient to specify min-el and max-el.